src/api/procedures/addInstruction.ts

```diff
@@ -69,6 +72,10 @@ export type Params = AddInstructionsParams & {
  */
 export interface Storage {
   portfoliosToAffirm: (DefaultPortfolio | NumberedPortfolio)[][];
+  /**
+   * TODO: WithMediator variants are expected in v 6.2. This check is to ensure a smooth transition and can be removed
+   */
+  withMediatorsPresent: boolean;
```

Comment:
I put this in so this wouldn't block our releases. Otherwise we'd need to make 6.2 a hard requirement in this PR